commands/operator-sdk/cmd/add/crd.go

```diff
@@ -60,26 +60,26 @@ func crdFunc(cmd *cobra.Command, args []string) {
 	verifyCrdFlags()
 	verifyCrdDeployPath()
 
-	fmt.Fprintln(os.Stdout, "Generating custom resource definition (CRD) file")
+	fmt.Println("Generating custom resource definition (CRD) file")
```

Comment:
Anyone have thoughts on whether we should keep `fmt.[F]?[Pp]rint*(...)` statements for binary step logging or switch to `log.Info*(...)`?

Reply:
Yes agree, think that would unify the logging, by switching to `log.Info()`. Although I am okay if that is done that in a separate PR.